lib/middleware/list.js

```diff
@@ -34,17 +34,17 @@ module.exports = function(req, next){
         var perm = project.rev ? (project.rev + " ").grey + project.domain : ""
         if (project.planName) {
           if (project.planName.indexOf("Standard") !== -1){
-            var pn = project.planName.grey  
+            var pn = project.planName.grey
           }else{
             var pn = project.planName.blue
           }
         }
 
         var row = [
           perm || project.domain,
-          project.timeAgoInWords.grey,
-          project.cmd.grey,
-          project.platform.grey,
+          project.timeAgoInWords ? (project.timeAgoInWords).gray : "",
+          project.cmd ? (project.cmd).gray : "",
```

Comment:
"gray --> grey" seems to be a typo
```suggestion
          project.timeAgoInWords ? (project.timeAgoInWords).grey : "",
          project.cmd ? (project.cmd).grey : "",
```